### PR TITLE
FH-3807 Update template for Template Service Broker

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

**Comment by aidenkeating** [comment]
@david-martin Would you mind taking a look? 

I'm not sure if those `status` attributes meant something that I might have missed or whether those template expose annotations in the secret were meant to be there. This leaves a template binding with one value, the public URL which sync is accessible at.

The metadata might need to be updated also, I just added some stuff.

File: fh-sync-server-DEVELOPMENT.yaml [hunk 1]
```diff
@@ -3,6 +3,17 @@ kind: Template
 metadata:
   creationTimestamp: null
   name: fh-sync-server
+  annotations:
+    openshift.io/display-name: FeedHenry Sync (Persistent)
+    description: FeedHenry Sync Server
+    iconClass: icon-nodejs
+    tags: instant-app,sync
+    template.openshift.io/long-description: Data Synchronisation by FeedHenry.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
```
Comment (david-martin) [inline fh-sync-server-DEVELOPMENT.yaml]: as this is upstream, does it make sense to have Red Hat? I would think FeedHenry would be OK here.

File: fh-sync-server-DEVELOPMENT.yaml [hunk 2]
```diff
@@ -343,27 +331,16 @@ objects:
...
-        status: "True"
-        type: Admitted
-      host: fh-sync-server-myproject.127.0.0.1.nip.io
-      routerName: router
-      wildcardPolicy: None
+
 - apiVersion: v1
+  kind: Secret
   data:
     database-admin-password: dUxrSmZQbXNqOFh4dmNpcg==
```
Comment (david-martin) [inline fh-sync-server-DEVELOPMENT.yaml]: probably a follow up issue, but we should generate these values rather than hardcoded

**Review by david-martin — APPROVED** [review]
This template was largely an 'oc export' of the resources after getting them running in an openshift cluster. Removing the unnecessary fields and exposed values makes sense

**Pull request merged**